extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -29,28 +29,7 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 		googleApi = new GoogleTag(),
 		hiddenSlots = [
 			'INCONTENT_LEADERBOARD'
-		],
-		recoveryInitialized = false;
-
-	function loadRecovery() {
-		if (recoveryInitialized) {
-			return;
-		}
-		log('SourcePoint recovery enabled', 'debug', logGroup);
-		recoveryInitialized = true;
-		googleApi = recoveryHelper.createSourcePointTag();
-		recoveryHelper.recoverSlots();
-	}
-
-	function loadSourcePoint() {
-		if (recoveryHelper.isBlocking()) {
-			loadRecovery();
-		} else {
-			recoveryHelper.addOnBlockingCallback(function () {
-				loadRecovery();
-			});
-		}
-	}
+		];
 
 	function collapseElement(element) {
 		slotTweaker.hide(
```

Comment:
Not sure if second argument is still needed in slotTweaker - then `recoveryHelper::isRecoveryEnabled` is not needed as well